static/app/views/dashboards/widgetBuilder/components/queryFilterBuilder.tsx

```diff
@@ -0,0 +1,188 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import Input from 'sentry/components/input';
+import {SearchQueryBuilder} from 'sentry/components/searchQueryBuilder';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconAdd, IconDelete} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {DisplayType} from 'sentry/views/dashboards/types';
+import {useWidgetBuilderContext} from 'sentry/views/dashboards/widgetBuilder/contexts/widgetBuilderContext';
+import {BuilderStateAction} from 'sentry/views/dashboards/widgetBuilder/hooks/useWidgetBuilderState';
+
+function WidgetBuilderQueryFilterBuilder() {
+  const {state, dispatch} = useWidgetBuilderContext();
+
+  const canAddSearchConditions =
+    state.displayType !== DisplayType.TABLE &&
+    state.displayType !== DisplayType.BIG_NUMBER;
+
+  const onAddSearchConditions = () => {
+    dispatch({
+      type: BuilderStateAction.SET_QUERY,
+      payload: state.query?.length ? [...state.query, ''] : ['', ''],
+    });
+  };
+
+  const renderQueryFields = () => {
+    return (
+      <div>
```

Comment:
We could probably just inline this all because there isn't much going on in this `renderQueryFields` function to separate out